extensions/secure-passport/v1/samples/python/src/secure_passport_ext/__init__.py

```diff
@@ -0,0 +1,117 @@
+from typing import Optional, Dict, Any, List, Callable
+from pydantic import BaseModel, Field, ValidationError, ConfigDict
+from copy import deepcopy
+
+# --- Extension Definition ---
+
+SECURE_PASSPORT_URI = "https://a2aprotocol.ai/ext/secure-passport/v1"
```

Comment:
```suggestion
SECURE_PASSPORT_URI = "https://a2a-protocol.org/ext/secure-passport/v1"
```

The URL you currently have is for a copycat A2A Protocol Docs site.